### PR TITLE
ESQL rule docs stub

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses a part of https://github.com/elastic/ux-writing/issues/41.

[Preview](https://security-docs_4012.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-esql-rule) - Added a new section that details how to create an ES|QL rule.

### CODE REVIEW

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -259,6 +259,42 @@ IMPORTANT: When checking multiple fields, each unique combination of values from
 For example, if a rule has an interval of 5 minutes, no additional look-back time, and a history window size of 7 days, a term will be considered new only if the time it appears within the last 7 days is also within the last 5 minutes. Configure the rule interval and additional look-back time when you <<rule-schedule, set the rule's schedule>>.
 . Click *Continue* to <<rule-ui-basic-params, configure basic rule settings>>.
 
+[discrete]
+[[create-esql-rule]]
```
Comment (vitaliidm) [inline docs/detections/rules-ui-create.asciidoc:263]: would we have a separate page dedicated to ES|QL later?

Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:263]: Sure, @vitaliidm! We can set one up for 8.11 or a future release. What do you have in mind for the separate page? Do you want to create a separate page that walks users through several types of ES|QL rule queries or maybe something different?

Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:263]: Quick update from Slack convo with Vitalii today: We're going to revisit this convo next Wednesday and will update the [tooltip text/link](https://github.com/elastic/ux-writing/issues/41#issuecomment-1747957148) if needed.

Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc]: Resolving because moved stub to new page that'll provide ample space for example queries.

**Comment by nastasha-solomon** [comment]
We decided to place the stub within the Create rule page and remove the new page that I created (ES|QL rule queries).

**Review by vitaliidm — APPROVED** [review]

File: docs/detections/rules-ui-create.asciidoc [hunk 2]
```diff
@@ -259,6 +259,37 @@ IMPORTANT: When checking multiple fields, each unique combination of values from
...
++
+For this type of query, use the operator `[metadata _id, _index, _version]` after defining the index source. This will allow for alerts to be deduplicated and linked to the source documents. For example:
++
+[esql]
+-----
+FROM logs* [metadata _id, _index, _version]
+| WHERE event.id == "test"
+| LIMIT 10
+-----
+
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:293]: Added in [4f9a0fa](https://github.com/elastic/security-docs/pull/4012/commits/4f9a0fa20ef809549e54b64b884f73ce70ba92c8).

**Pull request merged**